database/roles/moderator.sql

```diff
@@ -0,0 +1,10 @@
+-- Role: moderator
+-- Description: Moderator role with access to moderate content.
+INSERT INTO roles (name, can_login, is_admin) VALUES ('moderator', 1, 0)
+ON DUPLICATE KEY UPDATE name = VALUES(name), can_login = VALUES(can_login), is_admin = VALUES(is_admin);
+
+INSERT INTO grants (created_at, role_id, section, item, rule_type, action, active)
+SELECT NOW(), r_mod.id, 'role', NULL, 'allow', 'user', 1
```

Comment:
Role isn't specific enough, there are a lot of moderation actions which could apply, plus you could introduce some that we will need later here. @jules

Reply:
It also needs a sectional variant too.  @jules